addons/addon-base-raas/packages/base-raas-services/lib/storage-gateway/storage-gateway.js

```diff
@@ -19,7 +19,8 @@ const { runAndCatch } = require('@aws-ee/base-services/lib/helpers/utils');
 
```

Comment:
This isn't necessarily a part of your PR, should've caught it earlier. Services' naming convention is "<folderName>-service", so the name of this file should be "storage-gateway-service.js" (accordingly rename the corresponding test file)

Reply:
Ah! I actually talked about this with Jeet in the last PR. https://github.com/awslabs/service-workbench-on-aws/pull/180 Jeet suggested he envisions it to be a wrapper/library when the feature finishes, so named it without 'service' on purpose.

Reply:
Just updated the name to add 'service'